home-accountant-web/src/test/java/com/godeltech/ha/service/MavenVersionServiceTest.java

```diff
@@ -0,0 +1,18 @@
+package com.godeltech.ha.service;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+
+public class MavenVersionServiceTest {
+    private String currrentVersion = "1.1.0-SNAPSHOT";
+
+    @Test
+    public void getMavenVersionMethodShouldReturnCorrectVersion() {
+
+        MavenVersionService serviceTester = new MavenVersionService();
+        assertEquals(currrentVersion, serviceTester.getMavenVersion().toString());
```

Comment:
@MSapiton please use hamcrest assertions, follow the link
https://junit.org/junit5/docs/current/user-guide/#writing-tests-assertions-third-party